index.bs

```diff
@@ -125,6 +131,19 @@ Content-Type: application/json
 ```
 </div>
 
+<h4 id="redirect-response">Redirect Response</h4>
+
+The redirect destination MUST be the same registerable domain (eTLD+1) as the original request.
```

Comment:
Should there be a limit of how many redirects the provider should follow? Currently 1Password only follows a single redirect.